.pipelines/custom-templates/publish-template.yml

```diff
@@ -124,9 +131,9 @@ jobs:
           customCommand: 'pack'
 
       # Remove .npmrc file
-      - script: rm .npmrc
+      - script: rm ${{ parameters.path }}/${{ parameters.libName }}/.npmrc
         displayName: Remove .npmrc file
-        workingDirectory: '${{ parameters.path }}/${{ parameters.libName }}'
+        condition: eq('${{ parameters.libName }}', 'msal-angular')
```

Comment:
Do we want to do this for all libs?

Reply:
Other libs use the NPM task which uses a service connection instead of the .npmrc for authentication. We only do it for angular because it's deploy script has other steps and isn't published directly through the npm service connection (that's only configurable when using the NPM publish ADO task)

Reply:
I did have to add the check to .npmrc write step so it also only runs for angular though. Thanks!